MAINTAINERS.rst

```diff
@@ -1,8 +1,14 @@
-Notes for maintainers cutting a release.
+Notes for maintainers.
 
 Pre-release test plan
 ---------------------
 
+0. If no release branch exists, fork the version numbers in the release branch
+   (vX.Y-branch) and the master branch. See "Cutting a release branch", below,
+   for details.
+
+   The rest of these steps should be done in the release branch.
```

Comment:
I know what you mean, but should we make it even clearer ?
Switch to the newly created release branch:
```
git checkout vX.Y-branch`
```